compile.mk

```diff
@@ -16,10 +16,21 @@ SOURCE       = $(BUILDROOT)/Source
 LIBSOURCE    = $(BUILDROOT)/LibSource
 GENSOURCE    = $(BUILD)/Source
 TESTPATCHES  = $(BUILDROOT)/TestPatches
+DAISYSP      = $(BUILDROOT)/Libraries/DaisySP/Source
 CPPFLAGS += -I$(PATCHSOURCE)
 CPPFLAGS += -I$(LIBSOURCE)
 CPPFLAGS += -I$(GENSOURCE)
 CPPFLAGS += -I$(TESTPATCHES)
+CPPFLAGS += -idirafter $(DAISYSP)
+CPPFLAGS += -idirafter $(DAISYSP)/Control
+CPPFLAGS += -idirafter $(DAISYSP)/Drums
+CPPFLAGS += -idirafter $(DAISYSP)/Dynamics
+CPPFLAGS += -idirafter $(DAISYSP)/Effects
+CPPFLAGS += -idirafter $(DAISYSP)/Filters
+CPPFLAGS += -idirafter $(DAISYSP)/Noise
+CPPFLAGS += -idirafter $(DAISYSP)/PhysicalModeling
+CPPFLAGS += -idirafter $(DAISYSP)/Synthesis
+CPPFLAGS += -idirafter $(DAISYSP)/Utility
```

Comment:
Why do you use -idirafter here?

Reply:
That's due to their excellent taste in naming headers ;-)
https://github.com/electro-smith/DaisySP/issues/137
So including it with -I causes wrong file getting used in basicmath.c and a compilation error due to memset not getting found there.